docs/en/seatunnel-engine/hybrid-cluster-deployment.md

```diff
@@ -154,6 +154,27 @@ seatunnel:
 
 When `dynamic-slot: true` is used, the `job-schedule-strategy: WAIT` configuration will become invalid and will be forcibly changed to `job-schedule-strategy: REJECT`, because this parameter is meaningless in dynamic slots.
 
+### 4.7 Coordinator Service
+
+CoordinatorService responsible for the process of generating each job from a LogicalDag to an ExecutionDag, 
+and then to a PhysicalDag. It ultimately creates the JobMaster for the job to handle scheduling, execution, and state monitoring.
+
+**core-thread-num**
+
+The corePoolSize of seatunnel coordinator job's executor cached thread pool 
+
+**max-thread-num**
+
+The maximumPoolSize of seatunnel coordinator job's executor cached thread pool
```

Comment:
maybe use `the max job count can be executed at same time` is more clear to explane this.

user don't need to know what's coordinator

Reply:
> maybe use `the max job count can be executed at same time` is more clear to explane this.
> 
> user don't need to know what's coordinator

ok